EIPS/eip-7906.md

```diff
@@ -0,0 +1,272 @@
+---
+eip: 7906
+title: Restricted Behavior Transaction Type
+description: A transaction type that provides a way for senders to restrict the outcomes of their execution
+author: Alex Forshtat (@forshtat), Shahaf Nacson (@shahafn), Dror Tirosh (@drortirosh)
+discussions-to: https://ethereum-magicians.org/t/eip-restricted-behavior-transaction-type/23130
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-02-21
+---
+
+## Abstract
+
+This proposal allows the transaction payload to define write protections for balances and storage slots that are to be enforced on the consensus level.
```

Comment:
please don't insert line breaks by yourself unless introducing a new para, html render will auto render as per the screen size. similarly for the entire EIP